projects/example-app/src/app/auth/reducers/index.ts

```diff
@@ -42,7 +42,7 @@ export const selectLoggedIn = createSelector(selectUser, user => !!user);
 
 export const selectLoginPageState = createSelector(
   selectAuthState,
-  (state: AuthState) => state.loginPage
+  (state: AuthState) => state[fromLoginPage.loginPageFeatureKey]
```

Comment:
By seeing this PR, I think we can/should remove the type in these selectors?
```suggestion
  state => state[fromLoginPage.loginPageFeatureKey]
```

Reply:
Thanks for making this change!